src/main/java/rx/Observable.java

```diff
@@ -20,6 +20,7 @@
 import rx.functions.*;
 import rx.internal.operators.*;
 import rx.internal.producers.SingleProducer;
+import rx.internal.operators.OperatorOnBackpressureBuffer.OnOverflow;
```

Comment:
This enumeration should be part of a more public part of the library instead of the internal part.